### DISCUSSION TITLE
Capacitor cannot find Android SDK platform tools in custom location "ERR_SDK_PACKAGE_NOT_FOUND"

### DISCUSSION THREAD

**Discussion by nagychris** [ionic-team/capacitor#4828] [Q&A] [answered]
I wanted to run my Ionic/Capacitor app via `npx cap run android` on an Android emulator. Nevertheless, the process got killed with the following error:

```
× Deploying app-debug.apk to Pixel_3_API_30 - failed!
[error] ADBs is unresponsive after 5000ms, killing server and retrying...
    ERR_SDK_PACKAGE_NOT_FOUND: SDK package not found by location: 
    D:\Android_SDK_HOME\platform-tools.

    More details for this error may be available online:

            https://github.com/ionic-team/native-run/wiki/Android-Errors
```

The correct location of the platform tools would be: **D:\Android_SDK_HOME\Sdk\platform-tools**. My _ANDROID_SDK_HOME_ variable is set to **D:\Android_SDK_HOME**, but when I change it to **D:\Android_SDK_HOME\Sdk**, this still results in the same error.

Does anyone know how/from where Capacitor accesses the SDK location, and how I can configure it? Could not find any documentation about that in the Capacitor docs.

**Comment by jcesarmobile** [comment, marked as answer]
Capacitor uses native-run package to run the apps and the error contains a link to the docs.
It’s telling you that the platform-tools are not installed
https://github.com/ionic-team/native-run/wiki/Android-Errors#err_sdk_package_not_found

**Reply by nagychris** [reply]
Hi, as already stated in my question, i have a valid installation of the platform tools under _D:\Android_SDK_HOME\Sdk\platform-tools_, so this does not help me any further, unfortunately.

**Reply by jcesarmobile** [reply]
The variables Android uses are `ANDROID_HOME` (deprecated) or `ANDROID_SDK_ROOT`, not `ANDROID_SDK_HOME`, that's for android user data.

If you set any of those and open a new cmd window (cmd doesn't detect variable changes) it should work. It should include the `Sdk` in the path, so `D:\Android_SDK_HOME\Sdk` in your case.

**Reply by nagychris** [reply]
Thanks, appending the  `\Sdk` path to `ANDROID_SDK_ROOT` solved it for me 👍🏻
